research-domain-agent/src/test/java/care/smith/fts/rda/rest/it/GeneralIT.java

```diff
@@ -136,4 +137,37 @@ void callingStatusReturnsQueued() throws IOException {
                     .verifyThenAssertThat()
                     .hasOperatorErrors());
   }
+
+  @Test
+  void projects() {
+    StepVerifier.create(client.get().uri("/api/v2/projects").retrieve().bodyToMono(String.class))
+        .assertNext(
+            transferProcessDefinitions -> {
+              assertThat(transferProcessDefinitions).isEqualTo("[\"test\"]");
+            })
+        .verifyComplete();
+  }
+
+  @Test
+  void project() {
+    StepVerifier.create(
+            client.get().uri("/api/v2/projects/test").retrieve().bodyToMono(String.class))
+        .assertNext(
+            transferProcessDefinitions -> {
+              assertThat(transferProcessDefinitions).contains("trustCenterAgent");
+              assertThat(transferProcessDefinitions).contains("MII");
+              assertThat(transferProcessDefinitions).contains("fhirStore");
+            })
+        .verifyComplete();
+  }
+
+  @Test
+  void projectReturns404() {
+    StepVerifier.create(
+            client.get().uri("/api/v2/projects/doesNotExist").retrieve().bodyToMono(String.class))
+        .expectErrorMessage(
```

Comment:
Can we test on the response status being 404 here?

Reply:
Yes, but it's ugly. Have a look :)